kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -2023,6 +2023,16 @@ object KyuubiConf {
       .intConf
       .createWithDefault(65536)
 
+  val METADATA_SEARCH_WINDOW: OptionalConfigEntry[Long] =
+    buildConf("kyuubi.metadata.search.window")
+      .doc("The time window to restrict user queries to metadata within a specific period. " +
+        "For example, if the window is set to P7D, only metadata from the past 7 days can be " +
+        "queried. If not set, it allows searching all metadata information in the metadata store.")
```

Comment:
the doc should declare the affected user-faced interfaces, for example, it only affects xxx API, and with suggestions like, you may want to set this to XXX when XXX, also should include the side-effects if any.

Reply:
thanks, addressed